src/olympia/blocklist/admin.py

```diff
@@ -502,9 +502,19 @@ def url_link(self, obj):
         return format_html('<a href="{}">{}</a>', obj.url, obj.url)
 
     def block_history(self, obj):
+        submission = BlockSubmission.get_submission_from_guid(obj.guid).first()
```

Comment:
I assume from the `first()` that it's it possible that `get_submission_from_guid` could return more than one submission. Is it sufficient to only output the first pending submission found?

Reply:
yeah, it returns a queryset.  A guid should only be in one BlockSubmission at once (because adding it would otherwise have been blocked by the changes in this patch).  `.first()` is just an easy (lazy) shortcut because it will either return one `BlockSubmission` instance or `None`.